server/operator.go

```diff
@@ -406,7 +437,9 @@ func newTransferLeaderOperator(regionID uint64, oldLeader, newLeader *metapb.Pee
 }
 
 func (op *transferLeaderOperator) String() string {
-	return fmt.Sprintf("%+v", *op)
+	op.RLock()
+	defer op.RUnlock()
+	return jsonString(op)
```

Comment:
These `defer`s can be refactor to:
```
op.RLock()
res := jsonString(op)
op.RUnlock()
return res
```
Cost of `defer` in such small functions may be relatively big.